docs/package.json

```diff
@@ -18,6 +18,7 @@
     "@heroicons/react": "1.0.6",
     "@mdx-js/react": "^2.1.4",
     "@react-aria/ssr": "3.3.0",
+    "@react-spring/web": "^9.6.1",
```

Comment:
What's the cost of this extra dep? Does it automatically get treeshaken out on other pages?

Reply:
Just ran a build analysis before and after, and the difference is hardly noticeable, around 0.1kB additional.